doc/develop/getting_started/index.rst

```diff
@@ -148,8 +149,8 @@ The current minimum required version for the main dependencies are:
 
       #. `Install chocolatey`_.
 
-      #. Open a ``cmd.exe`` window as **Administrator**. To do so, press the Windows key,
-         type "cmd.exe", right-click the result, and choose :guilabel:`Run as
+      #. Open a ``cmd.exe`` terminal window as **Administrator**. To do so, press the Windows key,
+         type ``cmd.exe``, right-click the "Command Prompt" search result, and choose :guilabel:`Run as
```

Comment:
so is it just me or following these steps actually ends up opening the old cmd.exe terminal window? It does open windows terminal though if I launch the "Command Prompt" search result as a regular user.

Reply:
I think it is just you, on Windows 11:

![image](https://github.com/zephyrproject-rtos/zephyr/assets/12450381/8928533b-fd81-43a3-969c-1215a2595f44)

which is the Windows Terminal.

Reply:
Maybe here:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/12450381/adc2624a-15bf-4a9f-80f5-9f3af108518d)
See also https://www.makeuseof.com/make-windows-terminal-default-app/

Reply:
@kartben can you take another look?

Reply:
Hi... friend of mine who is technical was following the instructions and the first part of this about installing windows terminal sent him off into the weeds. He installed some windows terminal application and then went from there. 

I was also confused by this instruction. Why do we call it the windows terminal application when it is called the windows command prompt? Or are we trying to get someone to install something else?